index.bs

```diff
@@ -1019,7 +1019,9 @@ partial interface Window {
 };
 </xmp>
 
-The <dfn attribute for=Window>cookieStore</dfn> getter steps are to return [=/this=]'s [=relevant settings object=]'s {{CookieStore}} object.
+A {{Window}} has an <dfn attribute for=Window>associated CookieStore</dfn>, which is a {{CookieStore}}.
```

Comment:
I don't know that we want `attribute` here - are you following precedent somewhere?

Reply:
Ah sorry, just a combination of not knowing what I'm doing and trying to make the links work 🤦‍♀️ .
Thanks for the help!